maestrowf/maestro.py

```diff
@@ -183,13 +183,18 @@ def setup_argparser():
                                 help="Launch a study based on a specification")
     run.add_argument("-a", "--attempts", type=int, default=1,
                      help="Maximum number of submission attempts before a "
-                     "step is marked as failed.")
+                     "step is marked as failed. [Default: %(default)d]")
+    run.add_argument("-r", "--rlimit", type=int, default=1,
```

Comment:
Question: Where is this actually passed along to the conductor?

Reply:
Good catch -- I never pushed in the change. Corrected.